package.json

```diff
@@ -48,6 +48,7 @@
     "lzma": "^2.3.2",
     "node-fetch": "^2.6.1",
     "save-file": "^2.3.1",
+    "test": "^0.6.0",
```

Comment:
do we specifically need to add this dependency? Cause we use `mocha` as a test runner and this package seems to be another test runner